sanic/request.py

```diff
@@ -7,7 +7,11 @@
 try:
     from ujson import loads as json_loads
 except ImportError:
-    from json import loads as json_loads
+    from json import loads as _json_loads
+
+    def json_loads(i):
+        i = i.decode() if isinstance(i, bytes) else i
+        return _json_loads(i)
```

Comment:
Well as far as I can see it's only used by `Request.json` to parse the HTTP body, i.e. `self.body`, which should always be `bytes`. Or maybe I'm missing something? I'll leave that up to @channelcat.